src/prefect/environments/storage/local.py

```diff
@@ -0,0 +1,93 @@
+import cloudpickle
+import os
+from slugify import slugify
+from typing import Any, Dict, Iterable, List, TYPE_CHECKING, Union
+
+import prefect
+from prefect.environments.storage import Storage
+
+if TYPE_CHECKING:
+    from prefect.core.flow import Flow
+
+
+class LocalStorage(Storage):
+    """
+    Local Storage class.  This class represents the Storage
+    interface for Flows stored as bytes in the local filesystem.
+
+    Args:
+        - directory (str, optional): the directory the flows will be stored in;
+            defaults to `~/.prefect`
```

Comment:
What are your thoughts on a `~/.prefect/flows` subdirectory as default?

Reply:
Well actually then that would require creation of the directory

Reply:
Actually I like this idea, because it would give users a "safe" directory they could delete in one shot

Reply:
Yea I can do that; I can create the directory, no big deal.

Reply:
Updated -- good call!